src/App.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+import "./App.css";
+import Input from "./components/Input";
+import { Todo } from "./model";
+import TodoList from "./components/TodoList";
+import Header from "./components/Header"
+
+const App: React.FC = () => {
```

Comment:
import FC type from react like this `import React, { FC, useState } from "react"`